scripts/build-release-artifacts.sh

```diff
@@ -37,8 +38,12 @@ command -v helm >/dev/null 2>&1 ||
 
 ROOT_DIR="$(cd "$(dirname "$0")/.." ; pwd)"
 RELEASE_VERSION="${RELEASE_TAG:1:${#RELEASE_TAG}-1}"
+# This file name is used by the create github release script. If modifying,
+# also update it there as well.
+ASSETS_FILE="kubefed-${RELEASE_TAG}-asset-files.txt"
```

Comment:
Would it make sense to ensure git ignores these files?   And maybe ignore the release artifacts at the same time (and/or put both in an ignored directory e.g. release/<tag>/*)?

Reply:
I never do a fresh clone - I have confidence if my working tree is clean that I'm working from a good state.  What about that scenario is concerning to you?

Reply:
It's ensuring the working tree is clean without explicitly checking for ignored files and working outside the common `GOPATH`.